modules/bundle_example/main.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 func main() {
-	module.Main("bundle", []module.Module{
+	module.Main([]module.Module{
```

Comment:
allow bundle to have name?

Reply:
bundle name vs bindle inst name. like `lazyload` vs `lazyload_rev1`

Reply:
withdrawal of changes